tools/sof-ll-timer-check.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
```

Comment:
Most of what this script does is parsing a log file but there's neither "parsing" nor "log" in the name. This is a test code so I don't think this is an "implementation detail" that should be hidden.